### ISSUE TITLE
Скачивание ID сообществ

### ISSUE THREAD

**Issue by hdma** [kasthack/OSP#2]
Здравствуйте, уважаемый! Планируется ли добавление возможности скачивания списка айди сообществ, в которых состоит пользователь?

**Comment by kasthack** [comment]
* Это потребует отдельного запроса на каждого пользователя, что замедлит сбор в тысячу раз(т.е. сейчас собирается по тысяче юзеров за запрос).

* Текущий деплой вообще отвратительно работает -- у меня висит багрепорт в почте, который я поправил, но не выложил в downloads.

* Вообще, у меня был обратный скрипт -- который обходил сообщества и получал всех участников, причём по 25000 за запрос, что позволяло получить те же данные при полном обходе всей соцсети, но
   * это писалось и тестировалось 2-3 года назад 
   * как следствие, сейчас вряд ли работает

**Comment by hdma** [comment]
Наверно, замедление сбора непринципиально в случаях, когда скачивается не всё, а часть (по айди из определенных сообществ, регионов и т.п) — что расширяет возможности поиска человека и фиксирования одной из важнейших характеристик его интересов. Возможно, в перспективе Вы сочтёте возможным добавить такой сбор. Недавно скачал старую базу от 28.07.10 в подобном формате, с полями groupslist — [http://forum.script-coding.com/viewtopic.php?pid=128063#p128063](url) — но в этом разбитом архиве из 8 частей, отсутствуют второй и последний тома, отчего всё после первого тома недоступно. Еще собираюсь скачать, кажется, Вашу базу, которую Вы опубликовали — [https://ibit.to/torrent/users-database-1-264M--83fe72/](url)

Стало любопытно: как за один запрос можно получить инфу о 1000 юзерах? За счёт execute-процедуры, встроенной в запрос?

**Comment by hdma** [comment]
Уважаемый, не располагаете ли Вы наиболее крайней базой ВКонтакта, созданной до введения функции закрытия профилей?

**Comment by kasthack** [comment]
>Уважаемый, не располагаете ли Вы наиболее крайней базой ВКонтакта, созданной до введения функции закрытия профилей?

Нет, я давно этим не занимаюсь и даже личного профиля не осталось.

[Поговаривают](https://vc.ru/social/44750-vkontakte-razreshila-polzovatelyam-zakryvat-profili-odnoy-nastroykoy?comment=851850), что закрытие профиля не влияет на API, но я не проверял.

**Comment by kasthack** [comment]
>Стало любопытно: как за один запрос можно получить инфу о 1000 юзерах? За счёт execute-процедуры, встроенной в запрос?

users.get принимает на вход массив пользователей(до 1000 за раз), но при этом отдаёт не все поля.

Это описано в документации

**Comment by hdma** [comment]
Насчёт доступа к профилю через API, приходит ошибка: "Access denied: this profile is private".
